src/w3c/headers.js

```diff
@@ -243,14 +256,29 @@ export function run(conf) {
   if (conf.isUnofficial && !Array.isArray(conf.logos)) {
     conf.logos = [];
   }
+  if (conf.isUnofficial) {
+    if (conf.license && !licenses.has(conf.license)) {
```

Comment:
Should we check this regardless of `conf.isUnofficial`?

Reply:
yes, but I want to do that as a followup. I'm just interested in dealing with unofficial for now... eventually we need to rip out all the unofficial stuff from the W3C profile. None of that stuff should be here.